pyomo/core/plugins/transform/interval_tightener.py

```diff
@@ -0,0 +1,67 @@
+from pyomo.core.base.constraint import Constraint
+from pyomo.core.kernel.numvalue import value
+from pyomo.core.plugins.transform.hierarchy import IsomorphicTransformation
+from pyomo.repn.canonical_repn import generate_canonical_repn
+from pyomo.util.plugin import alias
+import logging
+import textwrap
+logger=logging.getLogger('pyomo.core')
+class BoundsTightener(IsomorphicTransformation):
+    """Tightens bound on constraint, by iterating through each variable and
```

Comment:
In my opinion, BoundsTightener is too general a name for this transformation. Bounds tightening is used in many difference contexts, including tightening the bounds on variables. How about "TightenConstraintBoundsFromVars" or something similar?